### PR TITLE
Add bios and profile pics

### PR DESCRIPTION
**Pull request by cansavvy**
Related to getting this website up to snuff. 

I've added the profile pics and bios that people gave to me. With some minor changes to try to make at least a little parallel.  It's not super parallel. You can tell the bios were written by each different person, but I think its okay to get started. We just need things on here. 

### Note for reviewers: 
- Note I had no idea what order to put people in so the order is Jeff and Sandy first because they are leadership/admin lead and then everyone else underneath "Members" is in alphabetical order. 
- I also changed `People` to `Our Team` because it sounds friendlier and more descriptive. 
- I just need a bigger profile pic as well as a bio from @vortexing 
- I also wrote a bio for @avahoffman so if she wants to check that she's okay with what I wrote, that'd be good. 
- The spelling errors are addressed on #21 

#### To preview: 
To preview this website, you can run jekyll locally run this: 
```
bundle exec jekyll serve
```
You may have to install things following this https://jekyllrb.com/tutorials/using-jekyll-with-bundler/ if you haven't run jekyll on your computer recently.

### CODE REVIEW

**Comment by avahoffman** [comment]
Thanks @cansavvy - added minor tweaks!

File: _pages/our_team.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    excerpt: 'Jeff is Chief Data Officer, Vice President, and J Orin Edson Foundation Chair of Biostatistics at the Fred Hutchinson Cancer Center. He leads the Fred Hutch Data Science Lab in building a data environment that is both cutting edge and friendly, supportive, open, and collaborative.'
+    url: "https://jtleek.com/"
+    btn_label: "More about Jeff"
+    btn_class: "btn--primary"
+amy:
+  - image_path: /images/profile-pics/amy-paguirigan.jpg
+    alt: "placeholder image"
+    title: "Amy Paguirigan - Senior Staff Scientist"
+    excerpt: 'Amy has worked at the Fred Hutch working to support bench and biomedical data scientists that develop novel methods to study intratumoral heterogeneity in cancer. She also led the creation of the [Fred Hutch Biomedical Data Science Wiki](https://sciwiki.fredhutch.org).'
+    url: "https://www.linkedin.com/in/amy-paguirigan-75a7299/"
```
Comment (cansavvy) [inline _pages/our_team.md:20]: @vortexing do you have a preferred link to a professional website or does LinkedIn work?

File: _pages/our_team.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    excerpt: "Dr. Wright’s work is focused on innovating ways to make data science and computational biology more accessible. She is a member of the Open Case Studies team, the [Genomic Data Science Community Network (GDSCN)](https://www.gdscn.org/), and the [Informatics Technology for Cancer Research (ITCR) Training Network (ITN)](https://www.itcrtraining.org/). She also currently leads the [ITCR Training and Outreach Working Group](https://www.itcrtraining.org/tow)."
+    url: "https://carriewright11.github.io/"
+    btn_label: "More about Carrie"
+    btn_class: "btn--primary"
+elizabeth:
+  - image_path: /images/profile-pics/elizabeth-humphries.jpg
+    alt: "placeholder image"
+    title: "Elizabeth Humprhies - Post-Doctoral Research Fellow"
+    excerpt: "Elizabeth is a postdoctoral fellow. She did her PhD in molecular epidemiology and her MS in evolutionary genetics. She works on the [AnVIL Project](https://anvilproject.org/)."
+    url: "https://www.linkedin.com/in/elizabeth-humphries-61202a103/"
```
Comment (cansavvy) [inline _pages/our_team.md:52]: @ehumph do you have a preferred professional website besides LinkedIn?

File: _pages/our_team.md (hunk 1 above)
Comment (vortexing) [reply _pages/our_team.md:20]: Linked in is fine and so is that blurb for now. Thanks!

Comment (cansavvy) [reply _pages/our_team.md:20]: Okay. Thanks. 👍

File: _pages/our_team.md (hunk 2 above)
Comment (ehumph) [reply _pages/our_team.md:52]: Nope, no website! Also, my last name has two letters swapped - it's Humphries

File: _pages/our_team.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    alt: "placeholder image"
+    title: "Carrie Wright - Senior Staff Scientist"
+    excerpt: "Dr. Wright’s work is focused on innovating ways to make data science and computational biology more accessible. She is a member of the Open Case Studies team, the [Genomic Data Science Community Network (GDSCN)](https://www.gdscn.org/), and the [Informatics Technology for Cancer Research (ITCR) Training Network (ITN)](https://www.itcrtraining.org/). She also currently leads the [ITCR Training and Outreach Working Group](https://www.itcrtraining.org/tow)."
+    url: "https://carriewright11.github.io/"
+    btn_label: "More about Carrie"
+    btn_class: "btn--primary"
+elizabeth:
+  - image_path: /images/profile-pics/elizabeth-humphries.jpg
+    alt: "placeholder image"
+    title: "Elizabeth Humprhies - Post-Doctoral Research Fellow"
```
Comment (cansavvy) [inline _pages/our_team.md:50]:
```suggestion
    title: "Elizabeth Humphries - Post-Doctoral Research Fellow"
```

**Comment by cansavvy** [comment]
I believe I've applied everyone's suggestions so I'm going to merge this.

**Pull request merged**